libbpf-rs/tests/test.rs

```diff
@@ -986,6 +986,14 @@ fn test_object_ringbuf_raw() {
     // Consume from a (potentially) empty ring buffer using poll()
     let ret = mgr.poll_raw(Duration::from_millis(100));
     assert!(ret >= 0);
+
+    // Call getpid multiple times, to refill the ring buffer.
+    unsafe { libc::getpid() };
+    unsafe { libc::getpid() };
+
+    // Consume exactly one item
+    let ret = mgr.consume_raw_n(1);
+    assert!(ret == 1);
```

Comment:
Should we also add a test making sure that with two events (or more) events available we also consume & report two? `1` really could mean so many things, so I'd think there is some value add.

Reply:
That's a good idea, I'll add a test for this case as well, thanks.